cmd/version.go

```diff
@@ -12,13 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package protoschema
```

Comment:
Why move this out of internal? Is this really something that should be part of exported API? Maybe it belongs in the `internal/cmd` package instead?

Reply:
I just didn't want it to use the protoschema package name. I moved it to 'internal'.